### PR TITLE
Fast integration tests: Show reconciler information in log

### PR DESCRIPTION
**Pull request by tillknuesting**
Changes proposed in this pull request:

"Improve fast-integration tests. When we get runtime status by kcp tool from KEB, please enhance that mechniasm and return status from reconciler by using kcp cli as well. That means I want to see runtime status from KEB and from reconciler  ( last operation , number of operations, info) every time after provisioning, deprovisioning, upgrade (kcp rc info -i {scheduling_id} -o json). 

Basically implementing these workflows and logging the results:

kcp reconciliations operations --shoot <shootName> -o json
kcp reconciliations info -i  <scheduling-id> -o json

### CODE REVIEW

File: tests/fast-integration/kcp/client.js [hunk 1]
```diff
@@ -101,6 +101,18 @@ class KCPWrapper {
     return JSON.parse(result);
   }
 
+  async reconciliations(query) {
+    let args = ['reconciliations', `${query.parameter}`, '--output', 'json'];
+    if (query.shootName) {
+      args = args.concat('--shoot', `${query.shootName}`);
+    }
+    if (query.shootName) {
+      args = args.concat('--scheduling-id', `${query.shootName}`);
```
Comment (ksputo) [inline tests/fast-integration/kcp/client.js:110]: shouldn't it be something like:

```suggestion
      args = args.concat('--scheduling-id', `${query.schedulingID}`);
```

File: tests/fast-integration/kcp/client.js [hunk 2]
```diff
@@ -101,6 +101,18 @@ class KCPWrapper {
     return JSON.parse(result);
   }
 
+  async reconciliations(query) {
+    let args = ['reconciliations', `${query.parameter}`, '--output', 'json'];
+    if (query.shootName) {
+      args = args.concat('--shoot', `${query.shootName}`);
+    }
+    if (query.shootName) {
```
Comment (ksputo) [inline tests/fast-integration/kcp/client.js:109]:
```suggestion
    if (query.schedulingID) {
```

File: tests/fast-integration/kcp/client.js [hunk 3]
```diff
@@ -156,6 +168,21 @@ class KCPWrapper {
     }
   };
 
+  async getReconciliationsOperations(shootName) {
+    await this.login();
+    const reconciliationsOperations = await this.reconciliations({parameter: 'operations',
+      shootName: shootName, ops: true});
```
Comment (ksputo) [inline tests/fast-integration/kcp/client.js:174]:
```suggestion
      shootName: shootName});
```

not needed

File: tests/fast-integration/kcp/client.js [hunk 4]
```diff
@@ -156,6 +168,21 @@ class KCPWrapper {
...
+    return JSON.stringify(reconciliationsOperations, null, '\t');
+  }
+
+  async getReconciliationsInfo(schedulingID) {
+    await this.login();
+    const reconciliationsInfo = await this.reconciliations({parameter: 'info',
+      schedulingID: schedulingID, ops: true});
```
Comment (ksputo) [inline tests/fast-integration/kcp/client.js:181]:
```suggestion
      schedulingID: schedulingID});
```
not needed

File: tests/fast-integration/skr-test/test.js [hunk 5]
```diff
@@ -33,6 +33,24 @@ describe('Execute SKR test', function() {
       const runtimeStatus = await kcp.getRuntimeStatusOperations(this.options.instanceID);
       console.log(`\nRuntime status after provisioning: ${runtimeStatus}`);
 
+      // kcp reconciliations operations -c <shootName> -o json
+      const objRuntimeStatus = JSON.parse(runtimeStatus);
+      const reconciliationsOperations = await kcp.getReconciliationsOperations(objRuntimeStatus.data[0].shootName);
+
+      // kcp reconciliations info -i <scheduling-id> -o json
```
Comment (ksputo) [inline tests/fast-integration/skr-test/test.js:40]: i'd move it below to the actual place of command execution

File: tests/fast-integration/skr-test/skr-test.js [hunk 6]
```diff
@@ -60,6 +60,24 @@ function oidcE2ETest() {
...
+
+      // getting last three operations
+      const lastObjReconciliationsOperations = objReconciliationsOperations.slice(objReconciliationsOperations.length-3,
+          objReconciliationsOperations.length);
+
+      let i;
+      for (i of lastObjReconciliationsOperations) {
+        const getReconciliationsInfo = await kcp.getReconciliationsInfo(i.schedulingID);
+        console.log(`\nReconciliation info for operation ${i.schedulingID}: ${getReconciliationsInfo}`);
+      }
```
Comment (ksputo) [inline tests/fast-integration/skr-test/skr-test.js:80]: think about extracting this duplicated logic to a helper function

**Review by wozniakjan — APPROVED** [review]
/lgtm
